spitfire/runtime/_baked.c

```diff
@@ -151,7 +186,7 @@ static PyTypeObject SanitizedPlaceholderType = {
   0,                                        /* tp_methods */
   0,                                        /* tp_members */
   0,                                        /* tp_getset */
-  0,                                        /* tp_base */
+  &PyUnicode_Type,                          /* tp_base */
```

Comment:
Since we're conditionally setting `tp_base` below, let's not set it here too
```suggestion
  0,                                        /* tp_base */
```